src/class/hid/hid_device.c

```diff
@@ -194,7 +194,9 @@ uint16_t hidd_open(uint8_t rhport, tusb_desc_interface_t const * desc_itf, uint1
 
   p_hid->boot_mode = false; // default mode is REPORT
   p_hid->itf_num   = desc_itf->bInterfaceNumber;
-  memcpy(&p_hid->report_desc_len, &(p_hid->hid_descriptor->wReportLength), 2);
+  
+  // Suppress IAR warning Pa039
```

Comment:
it can be tricky for people that doesn't use IAR. Maybe saying "Use offsetof to avoid pointer to the odd/misaligned address" would be better. I am not too picky, but we should have a space between castting and its operand e.g ` (uint8_t*) p_hid->hid_descriptor`